docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,66 @@
 ZWO Camera
 ==========
 
+This service operates a ZWO camera. The following are the different types of ZWO cameras that have been tested and used with catkit2 so far:
+
+- `ZWO ASI533MM <https://www.zwoastro.com/product/asi533mm-mc/>`_
+- `ZWO ASI290MM <https://agenaastro.com/zwo-asi290mm-cmos-monochrome-astronomy-imaging-camera.html>`_
+- `ZWO ASI178MM <https://agenaastro.com/zwo-asi178mm-cmos-monochrome-astronomy-imaging-camera.html>`_
+- `ZWO ASI1600MM <https://agenaastro.com/zwo-asi1600mm-p-cmos-monochrome-astronomy-imaging-camera-pro.html>`_
+
+For camera specs, see the website links above.
+Note that to use the ZWO cameras requires additional manual install of driver(s) from `zwoastro.com <https://astronomy-imaging-camera.com/software-drivers>`_ 
+
 Configuration
 -------------
 
+.. code-block:: YAML
+
+    camera1:
+        service_type: zwo_camera
+        simulated_service_type: camera_sim
+        requires_safety: false
+
+        device_name: ZWO ASI533MM
+        offset_x: 1038
+        offset_y: 1282
+        width: 192
+        height: 192
+        exposure_time: 1000
+        gain: 100
+
 Properties
 ----------
+``exposure_time``: Exposure time of the camera.
+
+``gain``: Gain of the camera.
+
+``brightness``: Brightness of the camera.
+
+``width``: The width of the camera.
```

Comment:
I just noticed that the description of width and height, as well as x and y offset are not really clear - it was me who wrote this for the Alvium and Hamamatsu camera but let's use this opportunity to fix this.

Width and height should be "of the camera frames." or "images" if you like that better.
Offset x and y should be: "... of the camera frames on the sensor." or anything that specifies better.

Could I also ask you to sneak in a fix for this in the Alvium and Hamamatsu service doc please?